src/main/resources/resource/layout/simplehtml.layout

```diff
@@ -4,5 +4,5 @@
 <dd>\format[AuthorLastFirst,AuthorLastFirstAbbreviator,AuthorAndsCommaReplacer,HTMLChars]{\editor} <i>(ed.)</i></dd>\end{editor}\begin{booktitle}
 <dd>\format[HTMLChars]{\booktitle}</dd>\end{booktitle}\begin{chapter}
 <dd>Chapter \format[HTMLChars]{\chapter}</dd>\end{chapter}
-<dd>\begin{journal}\format[HTMLChars]{\journal}, \end{journal}\begin{school}\format[HTMLChars]{\school}, \end{school}\begin{institution}\format[HTMLChars]{\institution}, \end{institution}\begin{publisher}\format[HTMLChars]{\publisher}, \end{publisher}\begin{howpublished}\format[HTMLChars]{\howpublished}, \end{howpublished}\begin{year}<b>\year</b>\end{year}\begin{volume}, Vol. \volume\end{volume}\begin{number}(\format[FormatPagesForHTML]{\number})\end{number}\begin{pages}, pp. \format[FormatPagesForHTML]{\pages}\end{pages}</dd>
+<dd>\begin{journal}\format[HTMLChars]{\journal}, \end{journal}\begin{school}\format[HTMLChars]{\school}, \end{school} \begin{publisher}\format[HTMLChars]{\publisher}, \end{publisher}\begin{howpublished}\format[HTMLChars]{\howpublished}, \end{howpublished}\begin{year}<b>\year</b>\end{year}\begin{volume}, Vol. \volume\end{volume}\begin{number}(\format[FormatPagesForHTML]{\number})\end{number}\begin{pages}, pp. \format[FormatPagesForHTML]{\pages}\end{pages}</dd>
```

Comment:
You sure about that additional whitespace after `\end{school}`?

Reply:
thanks! overlooked this